.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,22 @@
+- id: poetry-check
+  name: poetry-check
+  description: run poetry check to validate config
+  entry: poetry check
+  language: python
+  language_version: python3
+  pass_filenames: false
+- id: poetry-lock
+  name: poetry-lock
+  description: run poetry lock to update lock file
+  entry: poetry lock
+  language: python
+  language_version: python3
+  pass_filenames: false
+- id: poetry-export
+  name: poetry-export
+  description: run poetry export to sync lock file with requirements.txt
+  entry: poetry export
+  language: python
+  language_version: python3
+  pass_filenames: false
+  args: ["-f", "requirements.txt"]
```

Comment:
I thinka adding a `-o requirements.txt` makes sense here as default behaviour

Reply:
Yeah I think you are right.
My initial thought was that the command does not fail my default. But yeah I think `-o requirements.txt` will be my far the most usecase.

Changed it.

Reply:
Updated docs accordingly.